docker-compose.yml

```diff
@@ -35,10 +39,13 @@ services:
     env_file: .env
     ports:
       - 443:443
+      - 80:80
     volumes:
       - ${DOCKER_MOUNT_POINT}:/var/www/html:rw,delegated
       # Custom configuration
-      - ./nginx/servers/proximis.conf:/etc/nginx/conf.d/proximis.conf:ro
+      ###>volume-nginx<###
```

Comment:
Why not use an environment variable like what is configured for the sources mount point?